src/models/user.js

```
@@ -51,7 +51,7 @@ const userSchema = new Schema(
       country: { type: String },
       city: { type: String }
     },
-    photo: String,
+    photo: [{ url: { type: String }, publicId: { type: String } }],
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)